docs/contributing/04-prow-workflow.md

```diff
@@ -0,0 +1,88 @@
+
+For more information, refer to the [Kubernetes documentation](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md) regarding the OWNERS file as well as the [ANNOUNCEMENTS.md](https://github.com/kubernetes/test-infra/blob/master/prow/ANNOUNCEMENTS.md) file in the `kubernetes/test-infra` repository.
+
+## OWNERS_ALIASES file
+
+The OWNERS_ALIASES file contains a list of aliases that can be used in the OWNERS files instead of the GitHub usernames.
+It is useful when the same set of people is responsible for bigger amount of directories in the repo.
+
+The example structure of this file is as follows:
+```yaml
+aliases:
+  owners-group:
+    - githubUser1
+    - githubUser2
+    - githubUserN
+```
+
+For more information, refer to the official [Kubernetes documentation](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners_aliases).
+
+## Code review process
+
+Since the review process in Kyma is based on Prow, our flow is the same as described in the official [Kubernetes documentation](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process).
+
+## Commands
+
+Prow provides GitHub automations and ChatOps using `/foo` styled slash-commands. You can use these commands by adding a comment under a GitHub issue or pull request.
+
+
+This is a basic set of commands you need to know to manage issues and PRs in Kyma:
+
+|Command|Example|Description|Used by|Plugin|
+|---|---|---|---|---|
+|`/close`|`/close`|Close an issue or PR.|Authors and members of the organization |lifecycle|
+|`/reopen`|`/reopen`|Reopen an issue or PR.|Authors and members of the organization |lifecycle|
+|`/[remove-]lifecycle [stale/frozen/rotten/active]`| `/lifecycle frozen`, `/remove-lifecycle stale`|Command used to operate on lifecycle labels respected by [Kyma Stale Bot](https://github.com/apps/kyma-stale-bot).|anyone|lifecycle|
+|`/[un][remove-]hold [cancel]`|`/hold`, `/unhold`, `/remove-hold` `/hold cancel`|Adds or removes the `do-not-merge/hold` label used to indicate that the PR should not be automatically merged.|anyone|hold|
+|`/auto-cc`|`/auto-cc`|Requests review based on the OWNERS files if the reviewers were not assigned.|anyone|blunderbuss|
+|`/[un]cc [@username]`|`/cc @Ressetkk`, `/uncc`|Requests review from the specific person, or yourself.|anyone|assign|
+|`/[un]assign [@username]`|`/assign` `/unassign @Ressetkk`|(Un)assigns a person from an issue or PR.|anyone|assign|
+|`/[remove-](area,kind,priority,label,language) [name]`|`/area prow`, `/label Epic`, `/kind bug` `/remove-area ci`|Applies or removes a label from one of the recognized types of labels.|Anyone can trigger this command on issues and PRs. `triage/accepted` can only be added by org members. Restricted labels are only able to be added by teams and users in their configuration.|label|
```

Comment:
```suggestion
|`/[remove-](area,kind,priority,label,language) [name]`|`/area prow`, `/label Epic`, `/kind bug` `/remove-area ci`|Applies or removes a label from one of the recognized types of labels.|Anyone can trigger this command on issues and PRs. `triage/accepted` is a restricted label and can only be added by organization members. Only users that belong to at least one of the configured teams can add and modify the restricted labels.|label|
```

Reply:
^ maybe something like that then? 🤔

Reply:
`...can add and modify the restricted labels.` should be `...can use the restricted labels.` because it's up to the Prow maintainers to add the restricted labels to the plugin configuration

Reply:
I'm fine with your suggestion, changed 🙂